src/map/entities/battleentity.cpp

```diff
@@ -1426,6 +1426,12 @@ bool CBattleEntity::OnAttack(CAttackState& state, action_t& action)
     actionList_t& list = action.getNewActionList();
 
     list.ActionTargetID = PTarget->id;
+    
+    bool tredecim = false;
+	if (this->objtype == TYPE_PC && (((CCharEntity*)this)->getEquip(SLOT_MAIN))->getID() == 18052)
```

Comment:
It's not good to hard code specific item IDs in the core, it would be much better to handle this with an item mod